frontend/air-front-end/src/components/metrics/metric.component.css

```diff
@@ -0,0 +1,21 @@
+.chart-statistics-grid{
+    width: 100%;
+    height: 100%;
+    border-radius: 8px;
+    margin: 10px;
```

Comment:
the margin will trigger the overflow, pls remove it

this is caused by the parent container using viewport height & width, by which if you add margin into it it will increase the size so lets say height 100% (vh-100) + 10px